### ISSUE TITLE
Time resets back to morning everytime I die and respawn

### ISSUE THREAD

**Issue by sfaruque6** [rockhymas/iberia#45]
In the 1.12.2 edition, whenever I die and respawn, time resets back to 0, regardless of what time of the day it is. I removed all my other mods, yet the issue still persists.

**Comment by rockhymas** [comment]
Yeah, that was by design, so that you start your new spawn in the morning, and have a full day to set up protection for the night.

**Comment by sfaruque6** [comment]
Ok, but is there any way that I could remove that function so that time does not reset? I couldn't seem to find the config option for that.

**Comment by rockhymas** [comment]
Unfortunately, it doesn't look like. I'm not actively working on Iberia, but if you or someone else wants to fork the code and make changes, you're absolutely welcome to. If I come back to it at some point, I will keep this issue open and include it in the work I do, but it's honestly not on the radar right now. For you, or someone else, the code you'll want to modify is in https://github.com/rockhymas/iberia/blob/master/src/main/java/com/gibraltar/iberia/challenge/SpawnChallenge.java.

**Comment by sfaruque6** [comment]
I don't really have much knowledge in Java, so which part of the code should I alter in order to do so? I'm pretty sure it's between lines 211-226, unless if I should remove the whole thing by itself.

**Comment by rockhymas** [comment]
Yep, that would be the thing to remove.

**Comment by sfaruque6** [comment]
Ok, thanks. I just created a new pull request. How do I install it to the Iberia file for my Minecraft mods?
